core-bundle/src/Resources/config/controller.yaml

```diff
@@ -99,6 +99,12 @@ services:
 
     Contao\CoreBundle\Controller\FrontendModule\RootPageDependentModulesController: ~
 
+    Contao\CoreBundle\Controller\FrontendModule\FeedReaderController:
+        arguments:
+            - '@contao.feed.feedio'
+            - '@contao.feed.null_logger'
```

Comment:
any reason not to use our real logger?

Reply:
I wanted to silence the internals and only log exceptions catched at controller level. But on the other hand, if I log the internals, I could leave out logging the error in the controller.

Reply:
I don't know how the feed logger works or what level it logs to. If it generates a lot of noise for every page view and is only meant to be set for debugging purposes, maybe we should keep the `null_logger`.

Reply:
This seems strange to me. If the logger is optional, it should be provided as last argument and allow `null`. Then, when you don‘t want to log, you simply don‘t pass a logger instead of having to pass a null logger.

Reply:
Ok, now I saw what was wrong here: I only wanted to silence the logging from the library. But the comented code snippet is regarding the controller 🙈 This is oviously wrong, I wanted the real logger here oc.

Fixed in 5706e65d7